<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Gasolinera</class>
 <widget class="QWidget" name="Gasolinera">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>426</width>
    <height>317</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Gasolinera</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>10</y>
     <width>67</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Factura</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>90</y>
     <width>67</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Precio</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>210</y>
     <width>67</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Fecha</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>130</y>
     <width>101</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Km Recorridos</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>170</y>
     <width>67</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Litros</string>
   </property>
  </widget>
  <widget class="QDateEdit" name="c_fecha">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>206</y>
     <width>110</width>
     <height>31</height>
    </rect>
   </property>
   <property name="calendarPopup">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="sp_precio">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>80</y>
     <width>69</width>
     <height>27</height>
    </rect>
   </property>
   <property name="value">
    <double>50.000000000000000</double>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>80</y>
     <width>67</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Lps/litro</string>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="sp_km">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>120</y>
     <width>69</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>120</y>
     <width>51</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Km</string>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="sp_litros">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>160</y>
     <width>69</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>160</y>
     <width>67</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Litros</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>260</y>
     <width>99</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>Continuar</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
